src/fenicsxconcrete/sensor_definition/stress_sensor.py

```diff
@@ -34,17 +34,37 @@ def measure(self, problem: MaterialProblem) -> None:
             t : time of measurement for time dependent problems, default is 1
         """
         # project stress onto visualization space
+        stress, mandel_stress = None, None
         try:
             stress = problem.q_fields.stress
             assert stress is not None
         except AssertionError:
-            raise Exception("Stress not defined in problem")
+            # raise Exception("Stress not defined in problem")
+            pass
 
-        stress_function = project(
-            stress,  # stress fct from problem
-            df.fem.TensorFunctionSpace(problem.experiment.mesh, problem.q_fields.plot_space_type),  # tensor space
-            problem.q_fields.measure,
-        )
+        try:
+            mandel_stress = problem.q_fields.mandel_stress
+            assert mandel_stress is not None
+        except AssertionError:
+            # raise Exception("Mandel stress not defined in problem")
+            pass
```

Comment:
```suggestion
mandel_stress = problem.q_fields.mandel_stress
```
You test for None later and therefore we don't need the try block